shared/demo.js

```diff
@@ -0,0 +1,24 @@
+import ReactHelpers from './src/helpers/react';
+
+//require 'react-hot-loader/patch'
+// React = require 'react'
```

Comment:
it would be cool to have your script recognize commented code and remove it, but that seems hard